layouts/partials/head.html

```diff
@@ -97,6 +97,14 @@
     <!-- Our CSS bundle. -->
     <link rel="stylesheet" href="{{ getenv "REL_CSS_BUNDLE" }}">
 
+    <script src="https://browser.sentry-cdn.com/5.20.1/bundle.min.js" integrity="sha384-O8HdAJg1h8RARFowXd2J/r5fIWuinSBtjhwQoPesfVILeXzGpJxvyY/77OaPPXUo" crossorigin="anonymous"></script>
```

Comment:
Any chance we can load these scripts with `defer` or `async`?

Reply:
I believe in this case I'd want to _not_ defer, so as to be sure to catch errors as early as possible (which is why I placed it in the HEAD, above pretty much everything else). This is ~17K gzipped and forever-cached, though, so hopefully it won't register with the Googlebots.

Reply:
I'll look into this once more, though, just to see if there's a practical balance we can achieve.

Reply:
Yeah, given that we aren't deferring the component loader, and this is so cache-friendly already, I'd feel better if we didn't optimize for this just yet. We can always revisit if necessary, though.

Reply:
Ah, on second thought, we have a lot of includes that are not deferred, looks like. Let's keep this in mind and circle back to it as a more holistic PR.